fs_product_multi_image/views/product_category.xml

```diff
@@ -7,6 +7,7 @@
         <field name="name">product.category.form</field>
         <field name="model">product.category</field>
         <field name="inherit_id" ref="product.product_category_form_view" />
+        <field name="priority" eval="99" />
```

Comment:
Why do you put a specific priority. This could lead to error if others views with lower priority uses the image_ids fields as anchor to add new fields.

Reply:
Then I'm almost sure the image field is at the bottom of the form. If a view target the image_ids field, it should inherit from the view which add the field, not the base view.